app/src/main/res/layout/hint_navigation_bar.xml

```diff
@@ -24,6 +24,7 @@
             tools:ignore="ContentDescription"/>
 
     <ImageView
```

Comment:
nit: It'd be good to add a comment saying we set the image dynamically for reason X and rm the `src` tag so it's clearer.